### PR TITLE
Use KnativeEventing for the tracing, and patch it, not replace

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

### CODE REVIEW

File: openshift/e2e-tests.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ failed=0
 
 (( !failed )) && install_serverless || failed=1
 
-(( !failed )) && install_knative_eventing || failed=1
+##(( !failed )) && install_knative_eventing || failed=1
```
Comment (lberk) [inline openshift/e2e-tests.sh:21]: any reason we're commenting this out vs just removing?

Comment (matzew) [reply openshift/e2e-tests.sh:21]: ah, to test it ... :-)

**Pull request merged**